test/acceptance/tests.py

```diff
@@ -1201,6 +1263,30 @@ def test_feedback_only(self):
         self.staff_area_page.verify_learner_final_score("Final grade: 1 out of 1")
 
 
+@ddt.ddt
+class MultipleOpenAssessmentTest(OpenAssessmentTest, MultipeOpenAssessmentMixin):
+    """
+    Test the multiple peer-assessment flow.
+    """
+
+    def setUp(self):
+        super(MultipleOpenAssessmentTest, self).setUp('multiple_ora')
+        self.staff_area_page = StaffAreaPage(self.browser, self.problem_loc)
```

Comment:
Ah okay, I understand now.

Would you mind adding a comment to this line in the constructor? Something like "set up staff area page for both problems, they will be differentiated by vertical index numbers".